src/utilities/utility.ts

```diff
@@ -129,6 +137,75 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getFilePathsConfigs(): Promise<FileConfigPath> {
+
+    let filePathsConfig = <FileConfigPath>{};
+    const MANIFEST_PATHS_KEY = 'manifestFilePaths';
+    const HELM_CHART_KEY = 'helmChartFilePaths';
+    const DOCKERFILE_PATH_KEY = 'dockerfilePaths';
+    const DOCKERFILE_PATH_LABEL_KEY = 'dockerfile-path';
+
+    let inputManifestFiles = inputParams.manifests || [];
+    filePathsConfig[MANIFEST_PATHS_KEY] = inputManifestFiles;
+
+    let helmChartPaths = (process.env.HELM_CHART_PATHS && process.env.HELM_CHART_PATHS.split('\n').filter(path => path != "")) || [];
+    filePathsConfig[HELM_CHART_KEY] = helmChartPaths;
+
+    let imageNames = core.getInput('images').split('\n');
+    let imageDockerfilePathMap: any = {};
+    let pathValue: string, pathLink: string;
+    const branchOrTag: string = process.env.GITHUB_REF && process.env.GITHUB_REF.replace('refs/heads/','blob/').replace('refs/tags/','blob/');
+
+    //Fetching from image label if available
+    for (const image of imageNames) {
+        let args: string[] = [image];
+        let imageConfig: any;
+
+        try {
+            
+            await exec.exec('docker pull ', args, true).then(res => {
+                if (res.stderr != '' && !res.success) {
+                    throw new Error(`docker images pull failed with: ${res.stderr.match(/(.*)\s*$/)![0]}`);

```

Comment:
Is the flow expected to fail, if `docker pull` fails? 
I noticed you're relying on the `system` handling for errors. Should you just catch and display a warning instead?

Reply:
No it is throwing an error if pull fails and another if inspect fails (to show exactly why it failed), and then it is caught and warning it displayed in the next few lines, like you have suggested. Did I understand correctly?

Reply:
I think I missed the catch clause. My bad.

Reply:
Ok - resolving this then.